test/plugin/scenarios/undertow-worker-thread-pool-scenario/config/expectedData.yaml

```diff
@@ -22,29 +22,29 @@ meterItems:
           tags:
             - {name: metric_type, value: core_pool_size}
             - {name: pool_name, value: undertow_worker_pool}
-        singleValue: ge 1
+        singleValue: ge -1
```

Comment:
In new xnio implemention(3.8.0+), it return -1 if pool not ready or queue size is unlimited, so I change expectedData here.